src/main/java/frc/robot/subsystems/ElevatorIO.java

```diff
@@ -0,0 +1,31 @@
+package frc.robot.subsystems;
+
+import org.littletonrobotics.junction.AutoLog;
+
+public interface ElevatorIO {
+    
+    @AutoLog
+    public static class ElevatorIOInputs {
+
+        public double elevatorPosition = 0.0;
+        public double elevatorVelocity = 0.0;
+        public double leaderVoltage = 0.0;
+        public double leaderCurrent = 0.0;
+
+    
+        public double followerVoltage = 0.0;
+        public double followerCurrent = 0.0;
+    }
+
+  
+    public default void move(double volts){}
```

Comment:
Since you have a `move(double volts)` method and the motors are set to follow one another, you don't need `setLeaderVoltage()` or `setFollowerVoltage()`.